src/hotspot/share/memory/metaspace/metaspaceArena.cpp

```diff
@@ -241,7 +246,7 @@ MetaWord* MetaspaceArena::allocate(size_t requested_word_size) {
   }
 
   // Primary allocation
-  p = allocate_inner(requested_word_size);
+  p = allocate_inner(raw_word_size);
```

Comment:
Why wasn't this always raw_word_size which seems to be the aligned size based on the requested word size? ('aligned' might be a better word than 'raw' here)  If I'm reading it correctly.

Reply:
I think it was a plain oversight.